seqr_scripts/seqr_sample_qc.py

```diff
@@ -208,9 +204,11 @@ def run_platform_imputation(
     :return: Table with platform PCs and assigned platform
     :rtype: Table
     """
-    intervals = hl.import_locus_intervals(evaluation_intervals_path)
+    intervals = hl.import_locus_intervals(
+        "gs://gnomad-public/intervals/exome_evaluation_regions.v1.noheader.interval_list"
+    )
     callrate_mt = compute_callrate_mt(mt, intervals)
-    eigenvalues, scores_ht, _ = run_platform_pca(callrate_mt)
+    eigenvalues, scores_ht, ignore = run_platform_pca(callrate_mt)
```

Comment:
Curious about using a specific throwaway variable instead of the `_` - was there some issue using the underscore?

Reply:
In KCs review, I think she preferred seeing the ignore written out instead of using a symbol for it